js-exercises/minima/minima.test.js

```diff
@@ -0,0 +1,16 @@
+import { minima } from './minima';
+
+describe('minima', () => {
+  test('return the first k elements of the sorted array', () => {
+    expect(() => minima('', [5, 3, 4])).toThrowError(/[A-Za-z0-9]+/g);
+    expect(() => minima(2, [])).toThrowError(/[A-Za-z0-9]+/g);
+    expect(() => minima(2, [1])).toThrowError(/[A-Za-z0-9]+/g);
+    expect(() => minima(0, [])).toThrowError(/[A-Za-z0-9]+/g);
```

Comment:
I believe 
If someone says return me `0 elements`, then you should return empty array as empty array.
It will be make sense with positive test you wrote afterwards for `0 elements` input.